templates/appliance/index.html

```diff
@@ -151,60 +135,89 @@ <h2>App store easy</h2>
     <div class="col-5 u-align--center u-hide--small u-vertically-center">
       {{
         image(
-          url="https://assets.ubuntu.com/v1/ec3f53f7-extended-security-maintenance.svg",
-          alt="",
-          height="180",
-          width="180",
-          hi_def=True,
-          loading="auto"
+        url="https://assets.ubuntu.com/v1/ec3f53f7-extended-security-maintenance.svg",
+        alt="",
+        height="180",
+        width="180",
+        hi_def=True,
+        loading="auto"
         ) | safe
       }}
     </div>
     <div class="col-7">
-      <h2>Secure by design</h2>
-      <p class="p-heading--four">Crisp. Clean. Carrier-grade. Core.</p>
-      <p>SecureBoot. Full disk encryption. Hardware crypto keys. An ironclad commitment to your peace of mind.</p>
-      <p>We toughened Ubuntu to harden these things. They bring their magic, we bring our mettle. Community or business-backed, they both enjoy <a href="/core">the same robust, secure Ubuntu Core</a>.</p>
-      <p>Strict confinement and read-only packages keep bad code locked down. Certs and signatures make for tamper proof provenance. Full containerisation for no loose ends. And we’ve got your back for critical jobs with ten year updates, guaranteed.</p>
+      <h2>
+        Secure by design
+      </h2>
+      <p class="p-heading--four">
+        Crisp. Clean. Carrier-grade. Core.
+      </p>
+      <p>
+        SecureBoot. Full disk encryption. Hardware crypto keys. An ironclad commitment to your peace of mind.
+      </p>
+      <p>
+        We toughened Ubuntu to harden these things. They bring their magic, we bring our mettle. Community or business-backed, both enjoy the same robust, secure Ubuntu Core.
```

Comment:
"same robust, secure Ubuntu Core" is a link to /core in the copy doc

Reply:
removed from the copy doc